app/scripts/controllers/journalUpload.js

```diff
@@ -55,9 +55,7 @@ angular
       includeIncomplete: 'yes',
       primaryRegistrantOnly: true,
     };
-    $scope.meta = {
-      totalPages: 0,
-    };
+    $scope.meta = registrationsData.meta;
```

Comment:
So there was a bug that happened after I removed the duplicate api calls. Basically, we weren't updating ```$scope.meta``` until the user caused the query parameters to change and we ran ```refresh```. But we already had that metadata, so I just use it here on page load now.